lib.py

```diff
@@ -56,14 +56,19 @@ def confirm_update_token(ut, rt):
 # without sleeping)
 override_time_for_testing = False
 
-
+last_current_time = 0;
 def current_time():
+    global override_time_for_testing
+    global last_current_time
     if override_time_for_testing:
-        # TODO-DAN I commented out the next line - doesn't make sense to me with no parameters.
-        # logging
         return override_time_for_testing
     else:
-        return time.time()
+        t = time.time()
+        if t <= last_current_time + 0.00001:
+            t = t+0.00001
```

Comment:
gack, what a hack.... i think better to add some randomness or a serial number per time...